elements/pfe-tabs/src/pfe-tab-panel.scss

```
@@ -101,9 +113,14 @@ $LOCAL-VARIABLES: (focus: pfe-var(link),
   }
 
   :host([variant="earth"][vertical]) {
-    border-top: 0px;
-    border-left: pfe-var(ui--border-width) pfe-var(ui--border-style) pfe-var(surface--border);
+    border-top: pfe-local(BorderTop, $fallback: 0);
+    border-left: pfe-local(BorderLeft, $fallback: pfe-var(ui--border-width) pfe-var(ui--border-style) pfe-var(surface--border));
     height: 100%;
+    background-color: pfe-local(BackgroundColor, $region: panel, $fallback: #{pfe-var(surface--lightest)});
```

```suggestion
    background-color: pfe-local(BackgroundColor, $region: panel, $fallback: pfe-var(surface--lightest));
```

Should not require interpolation here either